CMakeLists.txt

```diff
@@ -531,6 +531,22 @@ if(WOLFSSL_AESCTR AND NOT WOLFSSL_FORTRESS)
         "-DWOLFSSL_AES_DIRECT")
 endif()
 
+# if we find an ARIA_DIR turn on ARIA support
+if ("$ENV{ARIA_DIR}" STREQUAL "")
```

Comment:
This check is kind of tricky. What if someone is trying to build without ARIA, doesn't supply the WOLFSSL_ARIA flag, but forgets about ARIA_DIR? I don't think having such an environment variable set should automatically turn on ARIA.

Reply:
My original plan was to turn on ARIA feature when the `ARIA_DIR` environment variable was found. I can see how that might be undesired.

I have a new solution in the works that will ignore the  `ARIA_DIR` if not enabled. Now the only way to enable is via Cmake flags, such as ` cmake .. -DWOLFSSL_ARIA=yes`

PR update soon.